app/config.js

```diff
@@ -13,6 +13,9 @@ config.api = {
 config.app = {
   env: process.env.ENV || 'dev',
   asset_path: '/static/',
+  protocol: process.env.APP_PROTOCOL || 'http',
+  host: process.env.APP_HOST || 'localhost',
+  port: process.env.APP_PORT || '3000',
```

Comment:
Should `port` be `process.env.APP_PORT || process.env.EXPRESS_PORT || 3000` ?

Reply:
Sure 👍